concordium-smart-contract-testing/src/impls.rs

```diff
@@ -1199,6 +1204,107 @@ impl Chain {
         }
     }
 
+    /// Invoke an external contract entrypoint.
+    ///
+    /// Similar to [`Chain::contract_invoke`](Self::contract_invoke) except that
+    /// it invokes a contract on the external node.
```

Comment:
Maybe emphasize this is dry running. Not sure `invoke` is enough.
```suggestion
    /// it invokes/dry runs a contract on the external node.
```